src/shared/types/record/statistics.ts

```diff
@@ -0,0 +1,8 @@
+export type Statistics = {
+  memberId: number;
+  email: string;
+  nickname: string;
+  recordStatisticsMap: {
+    [key: string]: number;
+  };
```

Comment:
key와 value가 각각 모두 같은 타입을 갖는 경우라면 Record를 사용하는 것이 더 깔끔한 것 같아요
```suggestion
  recordStatisticsMap: Record<string, number>;
```

Reply:
```
{
  "memberId" : 1,
  "email" : "abc@naver.com",
  "nickname" : "술로그123",
  "recordStatisticsMap" : {
    "막걸리" : 2,
    "기타" : 3,
    "과실주" : 5
  }
}
```
데이터가 위처럼 들어오는데 저렇게 바꿔도 가능한가요?

Reply:
관련 문서 찾아보시면 
```ts
{
   [key: string]: number;
}
```
이 구문과
```ts
Record<string, number>;
```
이 구문이 동일하다는 것 아실 수 있을 것 같습니다.

---

```ts
{
  "막걸리": 1,
  "기타": "123",
}
```
만약 이렇게 객체의 value 타입이 number와 string을 혼용한다면 1번방식이 편할 수 있는데
지금의 경우는 key, value가 string, number로 만 통일 되어 있어 Record가 더 명시적인것 같습니다!

Reply:
그렇군요 설명 감사합니다 🙏